pkg/syncer/fullsync.go

```diff
@@ -351,6 +355,7 @@ func fullSyncGetEntityMetadata(ctx context.Context, pvList []*v1.PersistentVolum
 		log.Errorf("getQueryResults failed to query volume metadata from vc. Err: %v", err)
 		return nil, nil, err
 	}
+	log.Debugf("FullSync: getQueryResults for volumeId %v clusterID%s  returns %v", queryVolumeIds, metadataSyncer.configInfo.Cfg.Global.ClusterID, allQueryResults)
```

Comment:
`clusterID%s` ==> `clusterID %s`

Reply:
`queryVolumeIds` is an array. Do we want to print that ?
If yes, then change `volumeId %v` to `queryVolumeIDs %v` or `volumeIDs %v`

Reply:
Yes. I remove the logs. And add debug logs in function "getQueryResults".